site/content/en/docs/concepts/preemption.md

```diff
@@ -54,11 +54,14 @@ Kueue offers two preemption algorithms. The main difference between them is the
 preemptions from a ClusterQueue to others in the Cohort, when the usage of the preempting ClusterQueue is
 already above the nominal quota. The algorithms are:
 
-- **[Classic Preemption](#classic-preemption)**: Preemption in the cohort only happens when the usage of the preempting ClusterQueue
-  will be under the nominal quota after the ongoing admission process, or when all the candidates for preemption belong to
-  the same ClusterQueue as the preempting Workload. In other words, ClusterQueues
-  can only borrow quota from others in the cohort if they do not preempt admitted Workloads from
-  other ClusterQueues. ClusterQueues in a cohort borrow resources in a first-come first-served fashion.
+- **[Classic Preemption](#classic-preemption)**: Preemption in the cohort only happens when:
+  - all the candidates for preemption belong to the same ClusterQueue as the preempting Workload 
+    or other borrowing ClusterQueues within the same cohort matching the preemptor's queue `borrowWithinCohort` policy.
+  - the usage of the preempting ClusterQueue will be under the nominal quota after the ongoing admission process
```

Comment:
Does usage have to be below nomial? If we have borrowWithinCohort, I think we can preempt while borrowing

https://github.com/kubernetes-sigs/kueue/blob/a128ae3354a56670ffa58695fef1eca270fe3a47/pkg/scheduler/flavorassigner/flavorassigner.go#L576-L583
https://github.com/kubernetes-sigs/kueue/blob/a128ae3354a56670ffa58695fef1eca270fe3a47/pkg/scheduler/flavorassigner/flavorassigner.go#L612-L613

Reply:
this refers to the quota reclaim scenario.  the ` if a.canPreemptWhileBorrowing() ..` is described above.

Reply:
You mean the list item on line 58? Got it. I think my confusion came from thinking the list items were an `AND`, rather than an `OR`

Reply:
done